File: baekjoon/11720.cpp
// 11720 :: 숫자의 합
#include <stdio.h>
int main() {
	int n;
	scanf("%d", &n);
	int sum = 0;
    
	while (n--) { // n이 0이 될 때까지 
        char x;
		scanf("%c", &x); // 숫자 하나씩 입력받음
		sum += x - '0';
	}
	printf("%d", sum);
	return 0;
}

File: baekjoon/11720.py
# 11720 : 숫자의 합
# C로 안풀려서 일단 파이썬으로 풀어보았다.
size = input()
number = input()

sum = 0
idx = 0

while idx < len(number):
    sum += int(number[idx])
    idx += 1

print(sum)
